mlflow/models/docker_utils.py

```diff
@@ -37,6 +37,20 @@
 RUN pip install virtualenv
 """
 
+if os.getenv("http_proxy") is not None and os.getenv("https_proxy") is not None:
+    MAVEN_PROXY = (
+        " -DproxySet=true -Dhttp.proxyHost={http_proxy_host} "
+        "-Dhttp.proxyPort={http_proxy_port} -Dhttps.proxyHost={https_proxy_host} "
+        "-Dhttps.proxyPort={https_proxy_port} -Dhttps.nonProxyHosts=repo.maven.apache.org"
+    ).format(
+        http_proxy_host=os.environ["http_proxy"].split("//")[1].split(":")[0],
+        http_proxy_port=os.environ["http_proxy"].rsplit(":", maxsplit=1)[1],
+        https_proxy_host=os.environ["https_proxy"].split("//")[1].split(":")[0],
+        https_proxy_port=os.environ["https_proxy"].rsplit(":", maxsplit=1)[1],
```

Comment:
Do we need exception handling for illegal http_proxy/http_proxy setting ?

Reply:
Will add assert statements to validate the proxy url.